File: LinguagemC++/EXERCICIO_CRESCENTE.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    //Lê dois números e informa se a ordem digitada é crescente ou decrescente
    //enquanto os números de entrada forem diferentes

    int firstNumber, secondNumber;
    
    cout << "Digite dois numeros: " << endl;
    cin >> firstNumber >> secondNumber;
    
    while(firstNumber != secondNumber)
    {
        if( firstNumber < secondNumber)
        {
            cout << "Crescente!" << endl;
        } else {
            cout << "Decrecente!" << endl;
        }
        
        cout << "Digite outros dois numeros: " << endl;
        cin >> firstNumber >> secondNumber;
    }

    return 0;
}

File: LinguagemPython/exercicioCrescente.py
# Lê dois números e informa se foi digitado na ordem crescente ou descreste

firstNumber: int
secondNumber: int

print("Digite dois números")
firstNumber = int(input())
secondNumber = int(input())

while firstNumber != secondNumber:
    if firstNumber > secondNumber:
        print("Decrescente!")
    else:
        print("Crescente!")

    print("Digite outros dois números")
    firstNumber = int(input())
    secondNumber = int(input())
